CRUD_operations.c

```diff
@@ -0,0 +1,131 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include<stdbool.h>
+typedef struct USER {
+    int id;
+    char name[50];
+    int age;
+} User;
+void createUser() {
+    User user;
+    FILE *file = fopen("user.txt", "a");
+    if (!file) {
+        printf("Error opening file\n");
+        return;
+    }
+    printf("Enter User ID: ");
+    scanf("%d", &user.id);
+    printf("Enter Name: ");
+    scanf("%s", user.name);
+    printf("Enter Age: ");
+    scanf("%d", &user.age);
+    fprintf(file, "%d,%s,%d\n", user.id, user.name, user.age);
+    fclose(file);
+    printf("User details created.\n");
+}
+void readUsers() {
+    FILE *file = fopen("user.txt", "r");
+    if (!file) {
+        printf("No users found.\n");
+        return;
+    }
+    char dest[100];
+    printf("\nID\tName\tAge\n");
+    printf("-------------------------\n");
+    while (fgets(dest, 100, file)) {

```

Comment:
take care of naming conventions for variables, functions and apply code formatting as suggested in calculator PR.